config/locales/idv/en.yml

```diff
@@ -44,19 +44,16 @@ en:
         please %{link}.
       activated_link: contact us
       hardfail: >
-        Your login.gov account can’t be used to access information with
-        participating agencies until you successfully complete this process.
+        We can't verify your identity right now.
       hardfail2: >
-        You can try to complete the verification process again in %{hours} hours.
+        You can try again in %{hours} hours.
       hardfail3: >
-        If you’re unable to verify your identity through this process there might
-        be other options available to you. Please see our troubleshooting page to learn about those options.
+        If you still can't verify your identity, visit our troubleshooting page 
+        to find answers to common questions about verification.
       hardfail4: >
-        You can also return to the agency website that directed you here to try to sign in there
-        for the services you are attempting to access.
+        You can also go to %{SP} for more help in accessing services.
```

Comment:
sure thing! this brings up that issue of sp vs no sp again -- if someone is signing into the IDP but were not coming from an SP, what do we want this to say? or should we just now show this message in that case?